src/Test/Trait/CrudTestIndexAsserts.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyCorp\Bundle\EasyAdminBundle\Test\Trait;
+
+// TODO : implement TODOs
+use function PHPUnit\Framework\assertCount;
```

Comment:
Pourquoi utiliser ce `use function` et pas avec tout les assertions ? Je pense qu'il vaut mieux l'enlever et utiliser `self::asertCount` partout par soucis d'homogénéité :smile: si il n'y a pas de raison technique (je peux passer à côté de quelque chose)

Reply:
oui, clairement... Merci PHPStorm pour l'ajout automatique et moi qui regarde à l'import... ;-)